src/types/Organization.ts

```diff
@@ -1,11 +1,14 @@
 import { ajv } from '../ajv';
 import type { JSONSchemaType } from 'ajv';
 import type { Writable } from './Writable';
+import type { ProposalFieldValue } from './ProposalFieldValue';
 
 interface Organization {
 	readonly id: number;
 	taxId: string;
 	name: string;
+	// TODO: use a type for values that covers either ProposalFieldValue or ExternalFieldValue.
```

Comment:
I think we can take out this TODO since we aren't using TODOs to manage project tasks and ExternalFieldValues don't exist right now.

What do you think of instead adding a comment on #1088 so that we don't lose this need?